### ISSUE TITLE
Issue with "Search on MDN"

### ISSUE THREAD

**Issue by alvinhui** [mdn/content#2616]
MDN URL: https://developer.mozilla.org/en-US/docs/MDN/Tools/Search

**What information was incorrect, unhelpful, or incomplete?**

Filtering results is not work.

**Specific section or headline?**

https://developer.mozilla.org/en-US/docs/MDN/Tools/Search#filtering_results

**What did you expect to see?**

If do not have this function, please delete the relevant content.

**Did you test this? If so, how?**

Look at this link: https://developer.mozilla.org/en-US/search?q=media+record

I didn't find the filter on this page

**Comment by peterbe** [comment]
CC'ing @Chris Mills 
Interesting. We dropped the “topic filtering” when we moved to React (~2 years ago?) but it stuck around on the Wiki. Now the Wiki is gone too. 
I think the answer to this issue is to remove a bunch of mentions of advanced search options. Is that something you can help with? I think it’s also a bit of a mistake to try to explain every little detail the site-search can do. For example, there’s a section  about the [“JSON response body format”](https://developer.mozilla.org/en-US/docs/MDN/Tools/Search#json_response_body_format). But this is an implementation detail that I don’t think we have any obligation to support perfectly. We have an obligation to display search results but the JSON is “our business”. It would be nice to have a public JSON API that we document and version and everything but the only reason we don't do that is lack of time and favoring of other features instead. 

Having said that, I happen to know that our search engine actually supports prefixes (e.g. search filtered specifically to the "Web/CSS/*" (for example) sub-tree). It's only implemented on the backend but it's nice because once [the new site-search UI](https://github.com/mdn/yari/pull/3009) launches, we can quite easily extend the UI with this advanced option.

**Comment by chrisdavidmills** [comment]
@peterbe ah right yes, I must've missed this page wen I went through and updated the MDN meta-docs. Is any of it worth keeping, or should I just delete the entire page for now?

**Comment by peterbe** [comment]
> @peterbe ah right yes, I must've missed this page wen I went through and updated the MDN meta-docs. Is any of it worth keeping, or should I just delete the entire page for now?

I can't imagine anything in there being useful. At least not until we have the time and energy to support a JSON API fully. For now, all a user needs to know is that you reach for search in the upper right-hand and type something and results show :)

**Comment by chrisdavidmills** [comment]
PR filed: https://github.com/mdn/content/pull/2657 feel free to merge it if you like ;-)
